styleguide/src/Forms/Dropdown/Dropdown.tsx

```diff
@@ -182,6 +183,8 @@ const DropdownComponent = (
         defaultValue={defaultValue}
         formatGroupLabel={(data) => formatGroupLabel(data, showGroupLength)}
         maxMenuHeight={maxMenuHeight}
+        menuPortalTarget={document.body}
```

Comment:
Acho que o document pode dar ruim quando for SSR.
`document?.body` deve resolver.

Reply:
só pra justificar, esse portal foi pra trabalhar junto com modal, o dropdown não tava sobrepondo o modal e fazia o modal expandir a height